Source/Urho3D/IO/File.h

```diff
@@ -137,6 +140,8 @@ class URHO3D_API File : public Object, public AbstractFile
 
     /// File name.
     ea::string fileName_;
+    /// Native file name.
+    ea::string nativeFileName_;
```

Comment:
Is there really a need for this? We have `GetNativePath()` in `FileSystem.h`.

Reply:
The problem is that `fileName_` _is not actually a file name_.
It is relative to resource cache directories and therefore worthless for my needs. Will add comment.